autopush/logging.py

```diff
@@ -108,6 +105,23 @@ def __call__(self, event):
             self._output.write(unicode(text))
             self._output.flush()
 
+    def raven_log(self, event):
+        f = event["log_failure"]
+        stack = extra = None
+        tb = f.getTracebackObject()
+        if not tb:
+            # include the current stack for at least some context
+            stack = list(iter_stack_frames())[4:]  # approx.
```

Comment:
Yea @bbangert. Of course your function could still be the reactor or Deferred's loop. We have a few of those in websocket.py -- addErrback(self.log_failure..). But at least now you would know PushServerProtocol.log_failure called it which is better than absolutely nothing